apps/challenges/models.py

```diff
@@ -220,3 +220,15 @@ class ChallengeConfiguration(TimeStampedModel):
     class Meta:
         app_label = 'challenges'
         db_table = 'challenge_zip_configuration'
+
+
+class StarChallenge(TimeStampedModel):
```

Comment:
Should we track an `unstar` action after a user has starred ?

If yes then tb schema needs to be updated.